src/ceph-disk/ceph_disk/main.py

```diff
@@ -1922,6 +1922,10 @@ def parser():
             help='unique OSD uuid to assign this disk to',
         )
         parser.add_argument(
+            '--crush-class',
```

Comment:
We already have **ceph osd crush class {ls,rm,create}** and it looked vaguely consistent to have **--crush-class**. But if it feels weird I can change it.